services/basic_types.proto

```diff
@@ -79,9 +79,30 @@ message AccountID {
     int64 realmNum = 2;
 
     /**
-     * A nonnegative account number unique within its realm
+     * The account number unique within its realm which can be either a non-negative integer or an alias public key.
      */
-    int64 accountNum = 3;
+    oneof account {
```

Comment:
I'm wondering, would it be better to make a wrapper type instead of changing `AccountID` directly?
```protobuf
message Account {
    oneof data {
        AccountID account_id = 1;
        Key key = 2;
    }
}
```
We can then deprecate `TranscationReceipt.account_id` and implement `TransactionReceipt.account` which would return that type.

Reply:
@danielakhterov Thank you ! 
Since we are using `alias` synonymous to ` accountNum`, (like `shard.realm.alias`) it need not be unique across different shard and realm. So, it is added inside `AccountID`.